crypto/err/err.c

```diff
@@ -667,6 +667,80 @@ ERR_STATE *ERR_get_state(void)
     if (!RUN_ONCE(&err_init, err_do_init))
         return NULL;
 
+    /*
```

Comment:
Is all that comment needed?  70 lines to explain a two-line if? :)

Reply:
I could make it part of the commit message instead and just have something like "avoid a memory leak" here. I want there to be *something* *somewhere*, so that a year or two from now, no one finds this call unnecessary.

Reply:
So I changed the comment to something shorter and moved the big explanation to the commit message.  Seems better?